mobile/client.go

```diff
@@ -163,6 +179,30 @@ func (ps *Props) GetObjectArray(key string) (ref *Objects, err error) {
 	return
 }
 
+func (ps *Props) EncryptToJWT(key, secret string, expire int64) (string, error) {
+	if len(key) == 0 || len(secret) == 0 || expire < 0 {
+		return "", errors.New("invalid parameters")
+	}
+
+	// Set "exp" field
+	ps.SetInt64("exp", expire)
```

Comment:
If the Props API has a means to set the expiration time, I don't see why it should be passed to this method as a special case.

(See also my contradictory comment in the SetInt64 method.)

Reply:
I can foresee some use case for the "expiration time" concept in general, but I don't see any other use cases for the `exp` key. We can extract it to a new method for `Props`, but then that would add an extra unnecessary step for generating JWE token.

Reply:
There is no need for a new method; SetInt64/SetFloat is already there.

Not forcing the use of the `exp` property makes EncryptToJWT more general. (Ideally I would also make it general over the JWT header, but that would cause unnecessary work for this use case.)

Reply:
True, we can use current `SetFloat` function. But my question is whether can we have a valid JWE token without `exp`? If not, then it would be always possible that the developer forgets to set it before generating the token. IMO, if things have to be done always in the same order, they should be in the same place.

Reply:
No payload fields are required by JWE.

Ninchat secure metadata requires `exp`, but on the other hand, EncryptToJWT doesn't enforce the `ninchat.com/metadata` property either. If Ninchat backend doesn't like the JWE, it will generate an error.

Currently the only purpose of EncodeToJWT is to enable easy testing where a JWE library is not otherwise available. Let's keep its implementation simple and API flexible.